guides/views.md

```diff
@@ -278,4 +278,12 @@ end
 
 ## Error pages
 
-Phoenix has a view called the `ErrorView` which lives in `lib/hello_web/views/error_view.ex`. The purpose of the `ErrorView` is to handle errors in a general way, from one centralized location.  Similar to the views we built in this guide, error views can return both HTML and JSON responses. See the [Custom Error Pages How-To](custom_error_pages.html) for more information.
+Phoenix has a view called the `ErrorView` which lives in `lib/hello_web/views/error_view.ex`. The purpose of `ErrorView` is to handle errors in a general way, from one centralized location.  Similar to the views we built in this guide, error views can return both HTML and JSON responses. See the [custom error pages how-to](custom_error_pages.html) for more information.
+
+
+[welcome page]: http://localhost:4000
```

Comment:
We don't use this style anywhere else in the guides. Can you please keep the links inline? Thank you.

Reply:
We just did
Please see the discussion for the reasoning.
https://github.com/phoenixframework/phoenix/pull/4202#pullrequestreview-588497696